### PR TITLE
adds sentry config

### PR DESCRIPTION
**Pull request by czpython**
Configure sentry when `SENTRY_DSN` environment variable is set.

### CODE REVIEW

File: aldryn_config.py [hunk 1]
```diff
@@ -191,6 +193,17 @@ def logging_settings(self, settings, env):
             }
         }
 
+    def sentry_settings(self, settings, env):
+        sentry_dsn = env('SENTRY_DSN')
+
+        if sentry_dsn:
+            settings['INSTALLED_APPS'].append('raven.contrib.django')
+            settings['RAVEN_CONFIG'] = {'dsn': sentry_dsn}
+            settings['LOGGING']['handlers']['sentry'] = {
```
Comment (Chive) [inline aldryn_config.py:202]: could this break if the user would overload `settings['LOGGING'] = None` ?

**Comment by Chive** [comment]
check my comment, LGTM

File: aldryn_config.py (hunk 1 above)
Comment (czpython) [inline aldryn_config.py:202]: user wouldn't be able to, this gets called as soon as logging is setup with self.logging_settings, once all of this runs then user is free to override anything on the settings.py in project

Comment (Chive) [inline aldryn_config.py:202]: okay, just making sure :) :+1:

**Pull request merged**